.github/workflows/ui-release-image.yaml

```diff
@@ -35,13 +48,59 @@ jobs:
         uses: docker/setup-qemu-action@v3
```

Comment:
You don't need `QEMU` anymore. It's for cross platform building.

Reply:
thanks for explanation!